packages/web/src/constants/google-login-props.ts

```diff
@@ -0,0 +1,3 @@
+export const clientId =
+  '480421853647-aihch4g2qqj5pndfadmdc9m7m7sts278.apps.googleusercontent.com';
```

Comment:
Maybe we should save sensitive data in env params?

Reply:
it is not sensitive it is appId. Made the same as for fb login.
appSecret will be in env file